src/ui/DynamicFacet/DynamicFacetRange.ts

```diff
@@ -44,6 +44,7 @@ export interface IDynamicFacetRangeOptions extends IDynamicFacetOptions {
  * - [`useLeadingWildcardInFacetSearch`]{@link DynamicFacet.options.useLeadingWildcardInFacetSearch}
  * - [`enableMoreLess`]{@link DynamicFacet.options.enableMoreLess}
  * - [`valueCaption`]{@link DynamicFacet.options.valueCaption}
+ * - [`sortCriteria`]{@link DynamicFacet.options.sortCriteria}
```

Comment:
Instead of listing the non-supported `IDynamicFacetOptions` options in the `DynamicFacetRange` component description, we could define an `ICommonDynamicFacetOptions` interface somewhere, and have both the `IDynamicFacetOptions` and `IDynamicFacetRangeOptions` interfaces extend that.

Reply:
🤦‍♂ that's very clever and so simpler!!

Reply:
https://github.com/coveo/search-ui/pull/1235/commits/15cf540d409e04f87fd7f83c21b85afc96819dd7